Installer/setup.iss

```diff
@@ -118,27 +119,34 @@ fr.AddCertDescription=Installer les certificats de {#MyAppSetupName}%nSi sélect
 de.AddCertDescription={#MyAppSetupName} Zertifikaten vertrauen%nDadurch erhalten Sie keine Warnungen mehr, wenn sich {#MyAppSetupName} aktualisiert.
 es.AddCertDescription=Confiar en los certificados de {#MyAppSetupName}%nDe esta forma no tendrá advertencias cada vez que {#MyAppSetupName} se actualice.
 it.AddCertDescription=Accetta i certificati di {#MyAppSetupName}%nIn questo modo non vedrai degli avvisi quando aggiornerai {#MyAppSetupName}.
+pt_br.AddCertDescription=Confiar nos certificados do {#MyAppSetupName}%nDeste modo não receberá alerta quando o {#MyAppSetupName} atualizar.
 en.ExistingSettings=Remove any existing settings
 fr.ExistingSettings=Supprimer les paramètres existants
 de.ExistingSettings=Alle vorhandenen Einstellungen löschen
 es.ExistingSettings=Elimiar cualquier configuración existente
 it.ExistingSettings=Rimuovi impostazioni esistenti
+pt_br.ExistingSettings=Remover configurações já existentes
 en.UninstallQuestion=Do you want to remove {#MyAppSetupName}'s settings?
 fr.UninstallQuestion=Voulez-vous aussi supprimer les paramètres de {#MyAppSetupName} ?
 de.UninstallQuestion=Sollen deine {#MyAppSetupName} Einstellungen gelöscht werden?
 es.UninstallQuestion=¿Quieres eliminar la configuración de {#MyAppSetupName}?
 it.UninstallQuestion=Vuoi rimuovere le impostazioni di {#MyAppSetupName}?
+pt_br.UninstallQuestion=Deseja remover as configurações do {#MyAppSetupName}?
 en.CertificatesGroup=Certificates:
 fr.CertificatesGroup=Certificats:
 de.CertificatesGroup=Zertifikate:
 es.CertificatesGroup=Certificados:
 it.CertificatesGroup=Certificati:
+pt_br.CertificatesGroup=Certificados:
 en.ViewReadmeFile=View the README file
 en.SupportTheProject=Support the project
 en.ViewChangelogFile=View the CHANGELOG file
 it.ViewReadmeFile=Visualizza file README
 it.SupportTheProject=Supporta il progetto
 it.ViewChangelogFile=Visualizza file CHANGELOG
+pt_br.ViewReadmeFile=Visualizar o arquivo README
+pt_br.SupportTheProject=Ajudar o projeto
+pt_br.ViewChangelogFile=Visualizar o CHANGELOG
```

Comment:
Please group these strings and add a blank line for better readability after each translated string like in products.iss:
```
en...
fr...
de...
es...
it...
pt...

en...
fr...
de...
es...
it...
pt...
```